components/generic/RelatedCollections.vue

```diff
@@ -4,21 +4,22 @@
     data-qa="related collections"
```

Comment:
This div (or its content) is not aligned is not aligned (left) with the title and description above. Or has this been fixed as part of the other PR?

Reply:
On collection pages, like http://localhost:3000/en/collections/topic/128-migration

Reply:
ok removed the padding